admin/app/football/controllers/FrontsController.scala

```diff
@@ -27,7 +27,12 @@ class FrontsController(val wsClient: WSClient, val mode: Mode.Mode) extends Cont
   def index = Action.async { implicit request =>
     for {
       competitions <- client.competitions.map(PA.filterCompetitions)
-      competitionTeams <- Future.traverse(competitions){comp => client.teams(comp.competitionId, comp.startDate, comp.endDate)}
+      competitionTeams <- Future.traverse(competitions){ comp =>
+        client.teams(comp.competitionId, comp.startDate, comp.endDate).recover {
+          // 'No data' is returned as an error by PA API. Therefore we ignore exception and replace it by an empty list
+          case e: Exception => List()
```

Comment:
Do we want to log/alert these exceptions somehow? I suppose we'll just get the heads up from central prod that there are competitions with empty lists anyway.

👍